serving/docs/lmi/user_guides/chat_input_output_schema.md

```diff
@@ -0,0 +1,259 @@
+# Chat Completions API Schema
+
+This document describes the API schema for the chat completions endpoints (`v1/chat/completions`) when using the built-in inference handlers in LMI containers.
+This schema is applicable to our latest release, v0.27.0.
+
+Chat Completions API supports streaming, and the response format for streaming differs from the response format for non-streaming.
+
```

Comment:
Can we add a couple sentences about how this works on sagemaker? something like:

```
On SageMaker, you can still send a chat completions request to the /invocations route. No additional configurations are needed. If your request contains the messages field, LMI will treat the request as a chat completions style request, and respond back with the chat completions response style.

This processing happens per request, meaning that you can support both our standard schema (link to schema), and chat completions schema with the same endpoint. 
```

Reply:
Added.